hr_employee_private_wizard/wizard/hr_employee_private_wizard.py

```diff
@@ -0,0 +1,103 @@
+# © 2020 - Today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from lxml import etree
+from odoo import api, models, fields, _
+
+WIZARD_FOOTER = """
+    <footer>
+        <button class="oe_highlight" name="save_and_close" type="object"/>
+        <button name="cancel" special="cancel"/>
+    </footer>
+"""
+
+
+class EmployeePrivateWizard(models.TransientModel):
+
+    _name = "hr.employee.private.wizard"
+    _description = "Employee Private Data Wizard"
+
+    employee_id = fields.Many2one("hr.employee")
+
+    def save_and_close(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5358386)